week-1-practise/level-3/1-piping.js

```diff
@@ -26,6 +25,14 @@ function multiply(a, b) {
 function format(num) {
   return "£" + num.toFixed(2);
 }
+var startingValue = 5;
+
+
+
+
+let result = format(add(multiply(2,5),10));
+               
 
+ multiply(5,2) | add(5,10) | format;
```

Comment:
What's this? :-)

Reply:
I was trying to use the piping sign ( |>) by it looks  this sign is not defined in vcode. Consequently, I used the brackets ways instead